docs/public/deploying-a-devfile-using-odo.adoc

```diff
@@ -91,42 +93,36 @@ In our example, we will be using `java-spring-boot` to deploy a sample https://s
 
 == Deploying a Java Spring Boot® component to an OpenShift cluster
 
-In this example we will be deploying an https://github.com/odo-devfiles/springboot-ex[example Spring Boot® component] that uses https://maven.apache.org/install.html[Maven] and Java 8 JDK.
+In this example we will be deploying a sample project associated with the java-spring-boot devfile that uses https://maven.apache.org/install.html[Maven] and Java 8 JDK.
```

Comment:
I don't agree that removing https://github.com/odo-devfiles/springboot-ex is good. We should have a link to the devfile that is being used.

Reply:
I removed it since we were using `--downloadSource`, but agreed actually.

Anyways, I'll add it back in when I revert back to using `git clone`